Packs/MicrosoftGraphDeviceManagement/Integrations/MicrosoftGraphDeviceManagement/MicrosoftGraphDeviceManagement.py

```diff
@@ -283,6 +285,20 @@ def get_managed_device_command(client: MsGraphClient, args: dict) -> None:
     return_outputs(human_readable, entry_context, raw_response)
 
 
+def get_managed_device_physical_memory_command(client: MsGraphClient, args: dict) -> None:
+    device_id: str = str(args.get('device_id'))
```

Comment:
Why do we need casting? arguments are always string

Reply:
fixed as agreed:
` device_id: str = args.get('device_id', '')`